api/v2/routes_rtfs.go

```diff
@@ -354,7 +354,7 @@ func (api *API) uploadDirectory(c *gin.Context) {
 	if err := z.Close(); err != nil {
 		// remove file if this fails
 		os.Remove(destPathZip)
-		Fail(c, err)
+		api.LogError(c, err, err.Error())(http.StatusInternalServerError)
```

Comment:
it might be safe to *not* return an error here - it's not relevant to the user whether the file was removed serverside, and we'll see the error in our logs

Reply:
I returned a generic error message, I don't think we can avoid aborting processing here, since it would mean that we aren't closing the read handler we have on the zip file.